docs/source/libraries/robocloud_items/index.rst

```diff
@@ -36,11 +36,15 @@ overriden manually while developing an activity.
 Item structure
 ==============
 
-A work item's payload is JSON and allows storing anything that is
+A work item's data payload is JSON and allows storing anything that is
 serializable. This library creates an object with the key 'variables'
 that contains key-value pairs of a variable name and its contents.
 These variables can be exposed to the Robot Framework task to be used directly.
 
+Parallel to the data section, a work item can also contain files,
```

Comment:
Parallel might be an unclear term here? Maybe just a boring "In addition ..." could work?

Reply:
A bit clunky, yes. Will fix.